src/components/NavBar/styles.scss

```diff
@@ -0,0 +1,119 @@
+@import "../commonStyles/constants.scss";
```

Comment:
can we set the text to use the same font? I understand not using the text constant here as it's a bit of different scenario, but the same font-family should be used thanks!